.pre-commit-config.yaml

```diff
@@ -49,3 +49,8 @@ repos:
     hooks:
     -   id: blacken-docs
         additional_dependencies: [black==20.8b1]
+-   repo: ../pre-commit-all-the-way-down/
```

Comment:
shouldn't it replace blacken-docs completely ?

Reply:
Anyway I see you used is with a local config, I except adding the hook here is actually temporary since the project may not be stable enough yet?